src/screens/Exchange/Exchange/Exchange.js

```diff
@@ -183,7 +218,10 @@ function Exchange() {
             disabled
             value={formattedToAmount}
             assetData={toAsset || fallbackAsset}
-            onAssetDataChange={(asset) => setToSymbol(asset.symbol)}
+            onAssetDataChange={(asset) => {
+              if (asset.chain !== chain) setChain(asset.chain);
```

Comment:
❗ 
This looks like a potential source of trouble, because if `to` asset changes `chain`, then `from` asset address from different chain no longer makes sense.

I see two options to solve it:
1) Keep `chain` state but restrict to options to that particular chain (which I believe was in the original design)
2) Keep `fromChain` and `toChain` separate states

Reply:
3. Reset `fromAsset` if `toAsset` changes state and vice versa.

Reply:
In case this happens (either `to` or `from` doesn't match) then `from` or `to` asset changes to native asset now –`fallbackAsset`.

Based on updated `ValueInput` component UI found in Zeplin and per what I discussed with Dmitry earlier in this case it should fallback not to `fallbackAsset` but to `Select asset`, but current `ValueInput` does not support that. 

The only way is either leave as is (falling back to `fallbackAsset`) or improve `ValueInput` so it would show `Select asset` label in case there's no asset provided (we can then remove `fallbackAsset` completely).